tests/test_arrow_dataset.py

```diff
@@ -2144,6 +2144,39 @@ def test_task_question_answering(self, in_memory):
                 )
                 self.assertDictEqual(features_after_cast, dset.features)
 
+    def test_task_automatic_speech_recognition(self, in_memory):
```

Comment:
don't hesitate to move it outside of the BaseDatasetTest class, as mentioned in #2529

Reply:
yep, will do this here as well!